packages/library-base/src/commonMain/kotlin/io/realm/kotlin/types/GeoCircle.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.kotlin.types
+
+/**
+ * This class represents a circle on the surface of a sphere. It cannot be persisted - i.e you
```

Comment:
```suggestion
 * This class represents a circle on the surface of the earth. It cannot be persisted - i.e you
```

It is not a generic sphere, isn't it?

Reply:
It is a generic sphere.

Reply:
Well, yes and no. I agree that `radians` will work on any sphere, but they are encapsulated inside the `Distance` class which can only convert to equatorial distances as the conversion to km and miles only make sense in the context of the earth.